.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,59 @@
+name: Bug report
+description: File a bug report
+title: '[BUG]'
+labels: bug
+
+body:
+  - type: markdown
+    id: description
+    attributes:
+      value: |
+          ## Before posting a bug report
+          Search existing GitHub issues to make sure the issue does not already exist:
+          https://github.com/xanaduai/pennylane/issues
+
+          For general technical details check out our documentation:
+          https://pennylane.readthedocs.io
+
+          # Issue description
+          Description of the issue - include code snippets and screenshots here if relevant.
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      description: |
+        What you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: actual
+    attributes:
+      label: Actual behavior
+      description: |
+        What actually happens?
+    validations:
+      required: true
+  - type: textarea
+    id: info
+    attributes:
+      label: Additional information
+      description: |
+        Please provide any additional information here. Configuration or data that might be necessary to reproduce the issue. This included system information (Post the output of `import pennylane as qml; qml.about()`) or reproducibility (What percentage of the time does it reproduce?).
+  - type: textarea
+    id: sourcecode
+    attributes:
+      label: Source code
+      description: Please include any additional code snippets related to the issue here. This will be automatically formatted into code, so no need for backticks.
```

Comment:
Instead of "additional", we should encourage "all code necessary to reproduce the error", or "minimal example that reproduces the error".

Reply:
I like "minimal non-working example".